cpp/.bazelrc

```diff
@@ -6,3 +6,13 @@ build --force_pic
 
 build --action_env=BAZEL_LINKOPTS=-static-libstdc++
 build --action_env=BAZEL_LINKLIBS=-l%:libstdc++.a
+
+build --noenable_bzlmod
```

Comment:
Can you help me understand what is this `--noenable_bzlmod`?

Reply:
This just disables bzlmod generation in favor of using the (legacy) WORKSPACE configuration: https://bazel.build/external/migration

I'll add a comment. If we were to exclude it, I believe Bazel would just create an empty `MODULE.bazel` file.

Reply:
Updated.